### DISCUSSION TITLE
Antenna Switch CTRL/ floating

### DISCUSSION THREAD

**Discussion by maxgerhardt** [Lora-net/LoRaMac-node#1393] [Q&A]
The [SX1261MBxAS](https://semtech.my.salesforce.com/sfc/p/#E0000000JelG/a/2R000000HUSF/0BIk9z8vpfnrPqx5xGq2WufNaJ3KsDrvLvK8TgO0ssg) board has a [PE4259 RF switch](https://www.psemi.com/pdf/datasheets/pe4259ds.pdf).

The implementation for this board sets the pin to analog in (`PIN_ANALOGIC`) when the switch is supposed to be turned off. 

https://github.com/Lora-net/LoRaMac-node/blob/24d55f6319b11104ba1ebb0317244418bb68bd81/src/boards/NucleoL152/sx1261mbxbas-board.c#L317-L325

Per datasheet the CTRL/ pin (that this piece of the software controls) is also the V_DD, positive supply voltage pin.

Does this pin not need to be driven to LOW instead of floating to prevent the switch from drawing any power?

**Comment by mluis1** [comment]
From our observations/testing letting this pin floating provides the expected result. 
You are free to keep it as `PIN_OUTPUT`

If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. In addition you may get better support by posting your question on [Semtech developer portal forum](https://forum.lora-developers.semtech.com/)
